src/assets/icons/Share.tsx

```diff
@@ -0,0 +1,32 @@
+function Share() {
+  return (
+    <svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <rect width="48" height="48" fill="#F4F4F400" />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M28.9091 16.5455C28.056 16.5455 27.3636 17.2378 27.3636 18.0909C27.3636
+           18.3405 27.4224 18.5746 27.5259 18.7817C27.7809 19.2902 28.3048 19.6364
+          28.9091 19.6364C29.7622 19.6364 30.4545 18.944 30.4545 18.0909C30.4545 
+          17.2378 29.7622 16.5455 28.9091 16.5455ZM25.8182 18.0909C25.8182 16.384 27.2021 
+          15 28.9091 15C30.616 15 32 16.384 32 18.0909C32 19.7979 30.616 21.1818 28.9091 
+          21.1818C27.978 21.1818 27.1442 20.77 26.5778 20.1201L21.1138 22.8525C21.1586 23.0619 
+          21.1818 23.2782 21.1818 23.5C21.1818 23.7218 21.1586 23.9381 21.1138 24.1475L26.5778 
+          26.8799C27.1442 26.23 27.978 25.8182 28.9091 25.8182C30.616 25.8182 32 27.2021 32 28.9091C32
+           30.616 30.616 32 28.9091 32C27.2021 32 25.8182 30.616 25.8182 28.9091C25.8182 28.6873 
+           25.8414 28.471 25.8862 28.2615L20.4222 25.5292C19.8558 26.179 19.022 26.5909 18.0909 
+           26.5909C16.384 26.5909 15 25.207 15 23.5C15 21.793 16.384 20.4091 18.0909 20.4091C19.022 
+           20.4091 19.8558 20.821 20.4222 21.4708L25.8862 18.7385C25.8414 18.529 25.8182 18.3127 
+           25.8182 18.0909ZM18.0909 21.9545C17.2378 21.9545 16.5455 22.6469 16.5455 23.5C16.5455 
+           24.3531 17.2378 25.0455 18.0909 25.0455C18.6944 25.0455 19.2191 24.6993 19.4741 24.1908C19.5776 
+           23.9837 19.6364 23.7496 19.6364 23.5C19.6364 23.2504 19.5776 23.0163 19.4741 22.8092C19.2191 
+           22.3007 18.6944 21.9545 18.0909 21.9545ZM28.9091 27.3636C28.3048 27.3636 27.7809 27.7098 27.5259 28.2183C27.4224 
+           28.4254 27.3636 28.6595 27.3636 28.9091C27.3636 29.7622 28.056 30.4545 28.9091 30.4545C29.7622 30.4545 30.4545 29.7622
+            30.4545 28.9091C30.4545 28.056 29.7622 27.3636 28.9091 27.3636Z"
+        fill="#131313"
+      />
+    </svg>
+  );
+}
```

Comment:
please use the approach for the use of the icons  read this documentation 
https://github.com/Ditmar/backoffice-openscience/blob/master/docs/icons.md